### PR TITLE
OP-22745: Build Medusa Docker image

### PR DESCRIPTION
**Pull request by c3-danmerino**

### CODE REVIEW

File: k8s/files/c3/cassandra/basic_test_v3.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+keyspace: stresscql
```
Comment (c3-amitsalunke) [inline k8s/files/c3/cassandra/basic_test_v3.yaml:1]: we should be packing this file

File: k8s/files/etc/medusa/medusa.ini [hunk 2]
```diff
@@ -0,0 +1,89 @@
+[cassandra]
```
Comment (c3-amitsalunke) [inline k8s/files/etc/medusa/medusa.ini:1]: How this file being used?

**Comment by c3-amitsalunke** [comment]
@c3-danmerino - Lets create epic branch to deliver all changes.

**Review by c3-amitsalunke — CHANGES_REQUESTED** [review]

File: k8s/files/etc/medusa/medusa.ini (hunk 2 above)
Comment (c3-danmerino) [reply k8s/files/etc/medusa/medusa.ini:1]: this is Medusa config. defines storage location, bucket, name of credential secrets and other options

File: k8s/files/c3/cassandra/basic_test_v3.yaml (hunk 1 above)
Comment (c3-danmerino) [reply k8s/files/c3/cassandra/basic_test_v3.yaml:1]: what do you mean by this?

**Pull request merged**